### PR TITLE
[luci-interpreter] Revise While kernel with copy method

### PR DESCRIPTION
**Pull request by seanshpark**
This will extract tensor copy to a method a remove similar repeating codes.

ONE-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
For #7385

File: compiler/luci-interpreter/src/kernels/While.cpp [hunk 1]
```diff
@@ -52,16 +73,7 @@ void While::execute() const
   const auto &cond_inputs = _cond_graph->getInputTensors();
   const auto &cond_outputs = _cond_graph->getOutputTensors();
 
-  // Copy initial kernel inputs to condition graph inputs.
```
Comment (seanshpark) [inline compiler/luci-interpreter/src/kernels/While.cpp:55]: Removed comments as the function call itself express what it does

File: compiler/luci-interpreter/src/kernels/While.cpp [hunk 2]
```diff
@@ -44,6 +44,27 @@ void While::configure()
   LUCI_INTERPRETER_CHECK(cond_outputs[0]->element_type() == DataType::BOOL);
 }
 
+void While::copy(const std::vector<const Tensor *> &src, const std::vector<Tensor *> &dst) const
```
Comment (binarman) [inline compiler/luci-interpreter/src/kernels/While.cpp:47]: Optional:
These methods could be simple static functions, they do not use any fields/methods of While.
In that case there are no need to change `While.h`.

**Review by binarman — DISMISSED** [review]
LGTM

File: compiler/luci-interpreter/src/kernels/While.cpp (hunk 2 above)
Comment (seanshpark) [reply compiler/luci-interpreter/src/kernels/While.cpp:47]: I'll follow your suggestion :)

**Review by binarman — APPROVED** [review]

**Pull request merged**